core/lib/types/src/l1/mod.rs

```diff
@@ -335,8 +397,7 @@ impl TryFrom<Log> for L1Tx {
             gas_per_pubdata_limit,
             op_processing_type: OpProcessingType::Common,
             priority_queue_type: PriorityQueueType::Deque,
-            eth_hash,
-            eth_block,
+            eth_block: 0,
```

Comment:
@RomanBrodetski @perekopskiy will it disrupt the work of eth-watch now? [This](https://github.com/matter-labs/zksync-era/pull/2067) adds a new way of remembering the last seen L1 block, but with this PR the current logic of the latest priority op will be broken

Reply:
Yes, it will. I suggest setting the actual `eth_block` here for now

Reply:
ack, reverted to setting the `eth_block` to `event.block_number`.